packages/algob/src/lib/tx.ts

```diff
@@ -266,16 +269,64 @@ export async function makeAndSignTx (
   return [txns, signedTxn];
 }
 
+/**
+ * Signs transaction object/s and returns raw signed transaction
+ * @param txn Transaction object/s
+ * @param signer signer
+ */
+export function signTransactionObject (
+  txns: Transaction | Transaction[], signer: wtypes.Signer | wtypes.Signer[]
+): Uint8Array | Uint8Array[] {
+  if (Array.isArray(txns)) {
+    txns = algosdk.assignGroupID(txns);
+    let signedTxn;
+    if (Array.isArray(signer)) {
+      signedTxn = txns.map((txn: Transaction, index: number) => {
+        return signTransaction(txn, signer[index]);
+      });
+    } else {
+      signedTxn = txns.map((txn: Transaction, index: number) => {
+        return signTransaction(txn, signer);
+      });
+    }
+    return signedTxn;
+  } else {
+    if (Array.isArray(signer)) {
+      throw new Error("cannot sign single transaction with multiple signers");
+    }
+    return signTransaction(txns, signer);
```

Comment:
I prefer this way, simpler for user

Reply:
I thought user would use `executeTransaction`. If we expose this to user, why not use an object?. 
Eg. `[{ txn: <Txn1>, signer: <sig1> }, { .. } ]`. 
Rather than using two arrays `([ txn1, txn2 ..], [sig1, sig2 ..])`.

The idea is signer is associated with a transaction, so it makes sense to pass them together. What do you think?

Reply:
ok